### PR TITLE
Update OpenTelemetry exporter link

### PR DESCRIPTION
**Pull request by jeanbisutti**
The previous link pointed to a Github file, not to a webiste

cc @brunobat

### CODE REVIEW

**Review by brunobat — APPROVED** [review]

File: docs/src/main/asciidoc/opentelemetry.adoc [hunk 1]
```diff
@@ -628,7 +628,7 @@ The exporter is automatically wired with CDI, that's why the `quarkus.otel.trace
 The `quarkus.otel.exporter.otlp.traces.protocol` default to `grpc` and `http/protobuf` can also be used.
 
 === On Quarkiverse
-Additional exporters will be available in the Quarkiverse https://github.com/quarkiverse/quarkus-opentelemetry-exporter/blob/main/README.md[quarkus-opentelemetry-exporter] project.
+Additional exporters will be available in the Quarkiverse https://docs.quarkiverse.io/quarkus-opentelemetry-exporter/dev/quarkus-opentelemetry-exporter-azure.html[quarkus-opentelemetry-exporter] project.
```
Comment (gsmet) [inline docs/src/main/asciidoc/opentelemetry.adoc:631]: Shouldn't it be https://docs.quarkiverse.io/quarkus-opentelemetry-exporter/dev/index.html instead?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/opentelemetry.adoc (hunk 1 above)
Comment (jeanbisutti) [reply docs/src/main/asciidoc/opentelemetry.adoc]: Yes! Sorry, a bad copy paste

**Review by gsmet — APPROVED** [review]
Thanks! I squashed the commits and force pushed.

**Comment by jeanbisutti** [comment]
> Thanks! I squashed the commits and force pushed.

Thank you!

**Comment by gsmet** [comment]
Thanks for taking the time to create a PR!

**Pull request merged**